bson/src/main/org/bson/BsonDocumentWrapper.java

```diff
@@ -34,12 +34,10 @@
  * @see org.bson.BsonDocumentWriter
  * @since 3.0
  */
-public class BsonDocumentWrapper<T> extends BsonDocument {
+public final class BsonDocumentWrapper<T> extends BsonDocument {
```

Comment:
Yes, though unlikely to affect anyone.  This is the down side of using BsonDocument like we do.  Ideally, all BsonValue subclasses would be final, but we can't make BsonDocument final, so the best we can do is make our subclasses of it final.